Describe 2FA for SSH (#484)
NethServer/docs: administrator-manual/en/base_system2.rst

3. generate a new code and copy it inside :guilabel:`Verification code` field, than click :guilabel:`Check code`
4. if the verification code is correct, click on the :guilabel:`Save` button

Two-factor authentication can be enabled for:

- the new Server Manager

> - SSH when using username and password (access with public key will never require 2FA)

Reviewer: ```suggestion
- SSH with password authentication 

.. notice:

    SSH public key authentication never asks for 2FA
```
